notifications/v1/types.go

```diff
@@ -0,0 +1,16 @@
+package v1
+
+// ⚠️ WARNING: These types MUST match the SAMS implementation, at
+// backend/internal/notification/types.go
+
+const (
+	nameUserDeleted = "UserDeleted"
+)
+
+// UserDeletedData contains information of a "UserDeleted" notification.
+type UserDeletedData struct {
+	// UserID is the SAMS external user ID of the deleted user.
```

Comment:
#namingishard

In working on SSC bits and integrating SAMS with other services, I've referred to this identifier a number of ways:

- SAMS external ID
- SAMS account external ID
- SAMS ID
- Account ID

... but I've personally never used "User ID" / "SAMS User ID". That isn't a good reason to suggest renaming this. But as an alternative, what do you think about `AccountID`. As in:

```go
// AccountID is the SAMS external ID for the deleted user's SAMS account.
```

Reply:
Fair! Gonna pick "AccountID" 😁